code/src/App.js

```diff
@@ -1,9 +1,70 @@
-import React from 'react'
+import React, { useState, useEffect }from 'react'
+import PostThoughts from "./Components/Post-Thoughts"
+import GetLastPost from "./Components/GetLast-Post"
+import { fecthData, postLikes, postThoughts } from './ServiceAPI/ApiServices'
 
 export const App = () => {
+
+  const [sendMessage, setSendMessage] = useState() 
```

Comment:
Potential nitpick, but personally I'd find "newPost" or "newMessage" more semantic of a name, as sendMessage intuitively sounds like a function rather than a variable to me

Reply:
taking notes thank you!!